JitsiConference.js

```diff
@@ -180,20 +186,18 @@ JitsiConference.prototype.addTrack = function (track) {
         var audioLevelHandler = this._fireAudioLevelChangeEvent.bind(this);
         track.addEventListener(JitsiTrackEvents.TRACK_MUTE_CHANGED, muteHandler);
         track.addEventListener(JitsiTrackEvents.TRACK_STOPPED, stopHandler);
-        track.addEventListener(JitsiTrackEvents.TRACK_AUDIO_LEVEL_CHANGED,
-            audioLevelHandler);
-        this.addEventListener(JitsiConferenceEvents.TRACK_REMOVED, function (track) {
-            track.removeEventListener(JitsiTrackEvents.TRACK_MUTE_CHANGED,
-                muteHandler);
-            track.removeEventListener(JitsiTrackEvents.TRACK_STOPPED,
-                stopHandler);
-            track.removeEventListener(
-                JitsiTrackEvents.TRACK_AUDIO_LEVEL_CHANGED, audioLevelHandler);
-        })
+        track.addEventListener(JitsiTrackEvents.TRACK_AUDIO_LEVEL_CHANGED, audioLevelHandler);
+        this.addEventListener(JitsiConferenceEvents.TRACK_REMOVED, function (someTrack) {
+            if (someTrack !== track) {
```

Comment:
We need that check because without it we will remove all event listeners from all local tracks after the first `JitsiConferenceEvents.TRACK_REMOVED` event